python/tests/test_file_format.py

```diff
@@ -69,22 +74,26 @@ def test_zero_items(self):
     def test_item_descriptor_format(self):
         for n in range(10):
             kas.dump(
-                {str(j): j * np.ones(j) for j in range(n)}, self.temp_file,
-                engine=self.engine)
+                {str(j): j * np.ones(j) for j in range(n)},
+                self.temp_file,
+                engine=self.engine,
+            )
             with open(self.temp_file, "rb") as f:
                 contents = f.read()
             self.assertEqual(struct.unpack("<I", contents[12:16])[0], n)
             offset = store.HEADER_SIZE
-            for j in range(n):
-                descriptor = contents[offset: offset + store.ITEM_DESCRIPTOR_SIZE]
+            for _ in range(n):
+                descriptor = contents[offset : offset + store.ITEM_DESCRIPTOR_SIZE]
```

Comment:
Thanks @benjeffery, this looks like the loop does the same thing in each iteration. I reckon there should be an `offset += store.ITEM_DESCRIPTOR_SIZE` after the `descriptor = ` line.

Reply:
Yep, well spotted @grahamgower. This test is currently just looking at the first descriptor each time.